.github/workflows/container-websvc.yml

```diff
@@ -0,0 +1,108 @@
+name: WebSvc Container
```

Comment:
@warrmr, can you please help me get the Docker container builds working? I don't think container.yml is working, I tried creating container-websvc.yml, but it's failing. I am not sure why and you seem to be the most knowledgeable. If you can help fix those in beta that would be great :)